ocp_resources/ocs_initialization.py

```diff
@@ -0,0 +1,35 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
```

Comment:
Missing a link to the resource doc

Reply:
There is no entry for this resource in the API doc

Reply:
as discussed, added a link to the source code